core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/store/dot-container-list.store.ts

```diff
@@ -0,0 +1,123 @@
+import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+import { MenuItem } from 'primeng/api';
+import { DotContainer } from '@models/container/dot-container.model';
+import { ActionHeaderOptions } from '@models/action-header';
+import { DataTableColumn } from '@models/data-table';
+import { ActivatedRoute } from '@angular/router';
+import { pluck, take } from 'rxjs/operators';
+
+export interface DotContainerListState {
+    containerBulkActions: MenuItem[];
+    selectedContainers: DotContainer[];
+    addToBundleIdentifier: string;
+    actionHeaderOptions: ActionHeaderOptions;
+    tableColumns: DataTableColumn[];
+    isEnterPrise: boolean;
+    hasEnvironments: boolean;
+}
+
+const defaultState: DotContainerListState = {
+    containerBulkActions: [],
+    selectedContainers: [],
+    addToBundleIdentifier: '',
+    actionHeaderOptions: {},
+    tableColumns: [],
+    isEnterPrise: false,
+    hasEnvironments: false
+};
+
+@Injectable()
+export class DotContainerListStore extends ComponentStore<DotContainerListState> {
+    constructor(private route: ActivatedRoute) {
+        super(defaultState);
```

Comment:
Take a look at this: https://ngrx.io/guide/component-store/initialization#lazy-initialization